libraries/shared/src/BlendshapeConstants.h

```diff
@@ -123,6 +113,24 @@ enum class LegacyBlendshpaes : int {
 // * LipsUpperOpen (not in ARKit)
 // * LipsLowerOpen (not in ARKit)
 
+const QMap<QString, QPair<QString, float>> BLENDSHAPE_SYNONYMS_MAP {
+    // ReadyPlayerMe
+    {"browDownLeft", {"BrowsD_L", 0.5f}},
+    {"browDownRight", {"BrowsD_R", 0.5f}},
+    {"browInnerUp", {"BrowsU_C", 0.3f}},
+    {"browOuterUpLeft", {"BrowsU_L", 0.3f}},
+    {"browOuterUpRight", {"BrowsU_R", 0.3f}},
+    {"eyeBlinkLeft", {"EyeBlink_L", 1.0f}},
+    {"eyeBlinkRight", {"EyeBlink_R", 1.0f}},
+    {"eyeWideLeft", {"EyeOpen_L", 1.0f}},
+    {"eyeWideRight", {"EyeOpen_R", 1.0f}},
+    {"mouthLeft", {"MouthSmile_L", 0.6f}},
+    {"mouthOpen", {"JawOpen", 1.0f}},
+    {"mouthRight", {"MouthSmile_R", 0.6f}},
+    {"mouthShrugLower", {"LipsUpperClose", 0.1f}},
+    {"viseme_O", {"LipsFunnel", 0.5f}}
```

Comment:
where do these float values come from?  is there documentation or is this from experimentation?

also, what if a different model standard has the same name but maps to a different blend shape or intensity?  do we have a way of saying “this was a RPM avatar, use this mapping”?

Reply:
> also, what if a different model standard has the same name but maps to a different blend shape or intensity? do we have a way of saying “this was a RPM avatar, use this mapping”?

I agree that this would have to be fixed to support other standards, but currently this is not a problem.
Other standards are not supported and will need to use FST files or change the models themselves. If someone uses an unsupported standard they are bound to have issues anyways. Unless of course there was an unsupported standard which used all of the ReadyPlayerMe blendshape names.

Reply:
Are you sure with this mapping?
I still think we can do better. If someone can confirm that we support all what is in the picture provided in the corresponding bug, we could do a better one.

Reply:
The values come from experimentation per issue https://github.com/vircadia/vircadia/issues/1168.

The ReadyPlayerMe glTF file doesn't contain an application name or similar explicitly indicating that it's from ReadyPlayerMe. (The only such name in it is "Blender".) However, I can wrap the synonym code in a condition that tests the presence of particular blendshape names - like done elsewhere for Mixamo files.

The blendshapes that Vircadia supports are (allegedly) those of ARKit. The actual list supported in the code is per: https://github.com/vircadia/vircadia/blob/master/libraries/shared/src/BlendshapeConstants.h I can add items to the mapping or change values if someone provides a better mapping.

Reply:
that sounds like a reasonable first step.  my other concern (discussed with Alezia) is why these magnitudes have to be reduced, because they are normal in blender.  this implies to me that we’re doing something wrong when loading them that messes up the magnitude